### ISSUE TITLE
UI: list users

### ISSUE THREAD

**Issue by MarcAntoine-Arnaud** [mcorbin/meuse#25]
In the small UI, it can be important for private to see list of users.
And maybe also the number of token generated for each user.

**Comment by mcorbin** [comment]
Currently, the UI is available to all users. I think listing users in the UI should be only available to admin. What do you think about ?

**Comment by MarcAntoine-Arnaud** [comment]
Yes I think too. But ideally it can be awesome if a configuration parameter can change that.
